tla/etcdraft.tla

```diff
@@ -103,7 +103,9 @@ VARIABLE
 VARIABLE 
     \* @type: Int -> Int;
     commitIndex
-logVars == <<log, commitIndex>>
+VARIABLE  
+    applied
```

Comment:
Missing type annotation

Reply:
TLC ignores these `@type: ...` type annotations, as they are only relevant to the Apalache model checker, which is not being used here.  As a matter of fact, they are not even part of the TLA+ language.  These annotations remain from the original upstream spec that was used as a basis for this spec.  I suggest removing all annotations.

Reply:
Agree with @lemmy , I remove all type annotations in the new commit.